certified-connectors/LCP - iCordis/Readme.md

```diff
@@ -0,0 +1,33 @@
+## LCP - iCordis
```

Comment:
@stijn-lcp please include all required section in the readme file. Please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).